sema4ai/pyproject.toml

```diff
@@ -40,7 +40,7 @@ pyscreeze = "^0.1.30"
 mss = "^9.0.1"
 
 java-access-bridge-wrapper = "^1.2.0"
-ruff = "^0.1.14"
+ruff = "^0.6.5"
```

Comment:
I think that this shouldn't be here (ruff should be a dev-only dependency).

Anyways, the PR seems good, we can remove it at a different PR since it was already there ;)